OAuthSwift/OAuth1Swift.swift

```diff
@@ -22,7 +22,7 @@ public class OAuth1Swift: OAuthSwift {
     var access_token_url: String
     
     // MARK: init
-    public init(consumerKey: String, consumerSecret: String, requestTokenUrl: String, authorizeUrl: String, accessTokenUrl: String){
+    public init(consumerKey: String, consumerSecret: String, requestTokenUrl: String, authorizeUrl: String, accessTokenUrl: String, paramsLocation : OAuthSwiftHTTPRequest.ParamsLocation){
```

Comment:
instead of adding new `init`, default value could be used
`, paramsLocation : OAuthSwiftHTTPRequest.ParamsLocation = .AuthorizationHeader`